### PR TITLE
Closes AgileVentures/MetPlus_tracker#280

### PR DESCRIPTION
**Pull request by patmbolger**

### CODE REVIEW

File: app/controllers/job_seekers_controller.rb [hunk 1]
```diff
@@ -33,26 +33,53 @@ def create
...
       redirect_to root_path
     else
-      @model_errors = @jobseeker.errors
       render 'new'
     end
   end
 
   def edit
     @jobseeker = JobSeeker.find(params[:id])
+    @current_resume = Resume.find_by_job_seeker_id(@jobseeker.id)
```
Comment (joaopapereira) [inline app/controllers/job_seekers_controller.rb]: The job seeker object format have access to this information?

**Comment by joaopapereira** [comment]
Generally looks good, check my comment
Looking at the controller we are starting to have a lot of logic there maybe we will need to rethink the approach and move the creation and update of the resume to the save of the job seeker model. Not saying note but in the future as an update

File: app/controllers/job_seekers_controller.rb (hunk 1 above)
Comment (patmbolger) [inline app/controllers/job_seekers_controller.rb]: Do you mean could I have done this instead?
```
@current_resume = @jobseeker.resumes[0]
```
then, yes, I could have.  Do you think that is a cleaner way to do it?  I am neutral on that.

**Comment by patmbolger** [comment]
I agree that the controller is looking kind of fat.  

Perhaps this is an argument for a model-level AS::Concern.  In fact, we may be able to reuse much of the same logic when we work on sending a Job to the cruncher (which in my mind right now will use a JobCruncher model as intermediary between Job model and CruncherService).

Something to work out later.

File: app/controllers/job_seekers_controller.rb (hunk 1 above)
Comment (joaopapereira) [inline app/controllers/job_seekers_controller.rb]: I think so because it hides the need of finding it under the hood of the JobSeeker

Comment (patmbolger) [inline app/controllers/job_seekers_controller.rb]: OK, I made that change.

**Comment by joaopapereira** [comment]
Looks good going to merge

**Pull request merged**